xml/libvirt_host.xml

```diff
@@ -1708,7 +1619,12 @@ done</screen>
           </listitem>
           <listitem>
            <para>
-            <guimenu>Source Path</guimenu>: The iSCSI target name (IQN).
+            <guimenu>Source IQN</guimenu>: The iSCSI target name (iSCSI Qualified Name).
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Initiator IQN</guimenu>: The iSCSI initiator name (iSCSI Qualified Name).
```

Comment:
Nice. :-) I think it is only necessary the 1st time, though... unless you disagree.